CONTRIBUTING.md

```diff
@@ -29,6 +29,37 @@ Push your changes up to your forked repository, then [create a new pull request]
 
 ## Style & Formatting
 
+### Tools
+The documentation includes ESLint and Prettier rules
+to enforce proper syntax and consistent formatting of JS and HTML files.
+ESLint and Remark enforce Markdown formatting rules.
+
+To apply the formatting rules as you write, install the ESLint package for your editor and make sure it is configured to use the `docs-v2/.eslintrc` config file.
+
+`git commit` runs the `lint-staged` pre-commit hook. The hook applies the rules in `docs-v2/.eslintrc` and reports syntax and style violations in staged Markdown, HTML, and JS files.
+For now, the commit hook doesn't validate the syntax of code blocks
+within Markdown files, e.g.
+
+  ````
+  ```js
+  function(...)
+  ```
+  ````
+
+You can run the lint scripts, defined in **package.json**, on demand with `yarn`.
+
+Check all files:
+
+```sh
+yarn run lint
+```
+
+Check Markdown style and grammar of `.md` files:
+
+```sh
+yarn run lint:md
+```
+
```

Comment:
Does this do the same thing as line 39? ...report syntax and style violations in staged files? If so, perhaps we can combine into procedural style, like:
Do one of the following:
- To check multiple files on demand
- To check/review files as you're writing